src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -551,8 +551,8 @@ JRT_LEAF(address, SharedRuntime::exception_handler_for_return_address(JavaThread
   return raw_exception_handler_for_return_address(current, return_address);
 JRT_END
 
-JRT_LEAF(jint, SharedRuntime::skip_value_scalarization(InlineKlass* klass))
-  return (jint)VectorSupport::skip_value_scalarization(klass);
+JRT_LEAF(int, SharedRuntime::skip_value_scalarization(InlineKlass* klass))
+  return (int) VectorSupport::skip_value_scalarization(klass);
```

Comment:
Thanks for addressing these, it will also fix the build issues with default options on Windows,  I was planning to address these with JDK-8311383